test/assertion/assert_equivalent_counts/test_assert_equivalent_counts.py

```diff
@@ -0,0 +1,85 @@
+import unittest
+
+from quantestpy import assert_equivalent_counts
+from quantestpy.exceptions import QuantestPyAssertionError
+
+
+class TestAssertEquivalentCounts(unittest.TestCase):
+    """
+    How to execute this test:
+    $ pwd
+    {Your directory where you git-cloned quantestpy}/quantestpy
+    $ python -m unittest \
+        test.assertion.assert_equivalent_counts.test_assert_equivalent_counts
+    ...
+    ----------------------------------------------------------------------
+    Ran 3 tests in 0.001s
+
+    OK
+    $
+    """
+
+    def test_exact_equivalent(self,):
+        counts_a = {
+            "00": 100,
+            "01": 0,
+            "10": 10,
+            "11": 3
+        }
+        counts_b = {
+            "00": 100,
+            "01": 0,
+            "10": 10,
+            "11": 3
+        }
+        self.assertIsNone(
+            assert_equivalent_counts(
+                counts_a,
+                counts_b,
+                sigma=1.
+            )
+        )
+
+    def test_approx_equivalent(self,):
```

Comment:
Why did not `test_approx_equivalent` give an error?
I think your definition of "tole" gives an error when key = "00":
```
abs(100-80) > sqrt(100) + sqrt(80)
```

Reply:
This is because the default value to `sigma` is `2`, and the following is satisfied:
```
abs(100-80) < 2 * (sqrt(100) + sqrt(80))
```